gems/sorbet-runtime/lib/types/private/methods/decl_builder.rb

```diff
@@ -6,6 +6,7 @@ module T::Private::Methods
 
   class DeclBuilder
     attr_reader :decl
+    attr_writer :raw
```

Comment:
ooc, what is this `raw` parameter? i don't quite see where it's used.

Reply:
also: is it intentional that this is only `attr_writer`?

Reply:
I think this was a bad rebase or something. We didn't need that line!